api/envoy/config/route/v3/route_components.proto

```diff
@@ -358,6 +358,14 @@ message Route {
   //    every application endpoint. This is both not easily maintainable and
   //    statistics use a non-trivial amount of memory(approximately 1KiB per route).
   string stat_prefix = 19;
+
+  // Sets the maximum size of a request body that will be buffered in memory. Envoy will return
+  // ``HTTP 413`` when buffer reaches the number set in this field.
+  // This option will be ignored if the current maximum buffer size is already larger than
+  // the value set in this field.
```

Comment:
it sounds like this option should be called `min_request_buffer_bytes`

Reply:
The name `min_request_buffer_bytes` may mislead users that Envoy will create a buffer with such a size from the beginning. In fact, it limits the max size that a buffer can go. I add "This option will be ignored ..." guard just because when the route is changed and a new smaller limit is applied, the behavior is unexpected  when a bigger buffer is already allocated.